code/datalab-app/src/app/components/welcome/DescribeElementSegment.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { withStyles } from 'material-ui/styles';
+
+const styles = theme => ({
+  outer: {
+    backgroundColor: theme.palette.secondary[50],
+  },
+  inner: {
+    maxWidth: 1024,
```

Comment:
I'm quite surprised that we need to specify this. Feels like something that the underlying layout should give to us.